sw/airborne/math/pprz_algebra_int.h

```diff
@@ -657,6 +665,20 @@ static inline void int32_vect_smul(int32_t *o, const int32_t *a, const int32_t s
   for (i = 0; i < n; i++) { o[i] = a[i] * s; }
 }
 
+/** Finds a value s in an array and returns the first location at which it is found, else -1 */
```

Comment:
I suggest to make the comment more accurate about the return value (true or false) and the value you get from the argument